app/src/page/firstRun/view.js

```diff
@@ -246,9 +289,11 @@ class FirstRunScreen extends React.PureComponent {
 
           <View style={firstRunStyle.buttonRow}>
             {([Constants.FIRST_RUN_PAGE_WELCOME, Constants.FIRST_RUN_PAGE_WALLET].indexOf(this.state.currentPage) > -1) && <View />}
-            {Constants.FIRST_RUN_PAGE_SKIP_ACCOUNT === this.state.currentPage &&
+            {(Constants.FIRST_RUN_PAGE_SKIP_ACCOUNT === this.state.currentPage ||
+             Constants.FIRST_RUN_PAGE_EMAIL_VERIFY === this.state.currentPage) &&
             <TouchableOpacity style={firstRunStyle.leftButton} onPress={this.handleLeftButtonPressed}>
-              <Text style={firstRunStyle.buttonText}>« Setup account</Text>
+              <Text style={firstRunStyle.buttonText}>
+                « {Constants.FIRST_RUN_PAGE_SKIP_ACCOUNT === this.state.currentPage ? 'Setup account' : 'Change Email'}</Text>
```

Comment:
Inconsistent casing. I think we generally do not capitalize the second word on actions like this, but what's important is just being consistent.